examples/undocumented/python/graphical/mclda.py

```diff
@@ -1,63 +1,57 @@
-from shogun import RealFeatures
 from shogun import MulticlassLabels
```

Comment:
could we have a `import shogun as sg` as per the issue I referenced?

Reply:
(and remove all the other imports?)

Reply:
Aye